### DISCUSSION TITLE
NEP Potential with other lattice dynamics packages

### DISCUSSION THREAD

**Discussion by rfhari** [brucefan1983/GPUMD#830] [Q&A] [answered]
Hi, I'm interested in using NEP potential to study thermal transport in AlGaN alloys. From the documentation and tutorials, I understand that NEP can be used for MD simulations to predict thermal conductivities using the Green-Kubo formalism or HNEMD.

But, I was wondering if NEP can interface with LAMMPS to calculate higher-order force constants and lattice thermal conductivity using other lattice dynamics packages like phono3py, alamode or shengbte? I’d appreciate your insights on this, as I couldn't find discussion s related to this.

**Comment by ZKC19940412** [comment, marked as answer]
Hi, GPUMD and NEP provide support to lattice dynamics such as kALDo (https://nanotheorygroup.github.io/kaldo/) and phono3py via its native python packages calorine or pynep. You can find an example using kALDo and NEP in https://github.com/dafolkner/Silicon_project/tree/main/kALDo/finite-difference and phono3py and NEP in https://gitlab.com/materials-modeling/calorine/-/blob/master/tutorials/thermal_conductivity_from_bte.ipynb?ref_type=heads

Hope they are helpful.

Best,

Zekun

**Reply by rfhari** [reply]
Hi Zekun, thanks for the quick reply! yeah, this is really helpful.
